src/Core/StackUri.php

```diff
@@ -205,13 +198,7 @@ private static function parseOptions(array $options)
         if (\count($optionKeys) !== \count($optionValues)) {
             throw new \InvalidArgumentException('The options given has to be an even array with key and value.');
         }
-        $combined = array_combine($optionKeys, $optionValues);
-        if (false === $combined) {
-            // returns false, if both are empty (or not equal length, but that's handled above)
-            // phpstan complained
```

Comment:
:-D now it complains when we do the check between false and $combined - probably it understands the code enough to realize we do the check.